### ISSUE TITLE
Analyzer: Slider not draggable

### ISSUE THREAD

**Issue by lolametro** [Sendouc/sendou-ink-mono#38]
## Steps to reproduce
- go to Analyzer page
- select Last Ditch Effort
- slider appears
- try to drag the slider with the mouse
### Expected
Slider moves
### Actual
The image on the slider moves, see screenshot (mouse is missing)
![image](https://user-images.githubusercontent.com/13728127/91638495-3fdc4d00-ea10-11ea-8518-499793c8f346.png)
## Hints
From a usability view, it's a bit hard to see this is a slider IMO. Maybe it's better to put the ability icon in front of the slider and keep the slider button a single color; so it looks like a slider.

**Comment by Sendouc** [comment]
Yeah need to think a bit what to do with this. I think the way I did it might have broken with Chakra UI 1.0 but not sure.
